sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -62,6 +62,10 @@ case class LocalRelation(output: Seq[Attribute], data: Seq[InternalRow] = Nil)
     case _ => false
   }
 
+  override def simpleString: String =
+    if (data == Seq.empty) super.simpleString + " [Empty Seq]"
```

Comment:
@marmbrus @cloud-fan Could you give a hint how to do it in `TreeNode`? 

It sounds like we are unable to determine if a node is empty in `TreeNode`. To know it, I think we have to check the value of `data` or `rows`? Thank you!